.gitignore

```diff
@@ -0,0 +1,11 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+*.pyc
+app/db.sqlite3
+db.sqlite3
```

Comment:
There is no need to add two lines to ignore the same DB file. Rewrite this in a way to ignore all files with `.sqlite3` extension regardless of their name or placement